src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+  width=600,
+  height=909,
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+IMG, DIV, P, and H1 made up 83% of all LCP nodes (on mobile). This doesn't tell us if the content was an image or text, as background images can be applied with CSS.
+
+{{ figure_markup(
+  image="performance-lcp-with-image.png",
+  caption="LCP elements with images, by device",
+  description="Bar chart showing 79% of desktop pages have an LCP element with an image and 71% of mobile pages",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=992354142&format=interactive",
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+We can see that 71-79% of pages had an LCP element that was an image, regardless of HTML node. Furthermore, desktop devices had a higher rate of LCPs as images. This could be due to less real estate on smaller screens pushing images out of the viewport resulting in heading text being the largest element.
+
+In both cases, images comprised the majority of LCP elements. This warrants a deeper dive into how those images are loading.
+
+{{ figure_markup(
+  image="performance-lcp-antipatterns.png",
+  caption="LCP elements with performance anti-patterns",
+  description="Bar chart showing 9.3% of pages on mobile use native lazy load on the LCP element, 16.5% probably lazy load, and 0.4% use async decode.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=130632749&format=interactive",
+  sheets_gid="730149797",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+For user experience, we want LCP elements to load as fast as possible. User experience is why LCP was selected as one of the Core Web Vitals. We do not want it to be lazy-loaded nor decoded asynchronously as that further delays the render. However, we can see that 9.3% of pages used the native loading=lazy flag on the LCP `<img>` element.
+
+Not all browsers support native lazy loading. Popular lazy loading polyfills detect a "lazyload" class on an image element. Thus, we can identify more possibly lazy-loaded images by adding images with a "lazyload" class to the total. The percent of sites probably lazy loading their LCP `<img>` element jumps up to 16.5% on mobile.
+
+Lazy loading your LCP element will result in worse performance. *Don't do it!* WordPress was an early adopter of native lazy loading. The early method was a naive solution applying lazy loading to all images, and the results showed a <a hreflang="en" href="https://web.dev/lcp-lazy-loading/">negative performance correlation</a>. They were able to use this data to implement a more nuanced approach for better performance.
+
+Luckily, only 0.4% of sites used the async decode directive for their LCP image. The negative impact of asynchronous decode is not as high as lazy loading.
```

Comment:
Why is this bad?

I really don't get this attribute, but here was my latest understanding which suggests it SHOULD be used: https://twitter.com/tunetheweb/status/1394683568874475522?s=20

Reply:
I'm re-reading that chapter in Addy's book, and I still think it might be the wrong decision for the LCP image though I have no data to back it up. Would it be possible to get Addy's input? I understand why you might want it for other images I just have my doubts on the LCP element. @addyosmani